Fix intersphinx_mapping config syntax for Sphinx doc generation.

Upstream issue: https://github.com/pimutils/khal/issues/1356

```
ERROR: Invalid value `None` in intersphinx_mapping['http://docs.python.org/'].
Expected a two-element tuple or list.
```

--- khal-0.11.3-origin/doc/source/conf.py
+++ khal-0.11.3/doc/source/conf.py
@@ -309,4 +309,4 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
